<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Jake Yeung </title> <meta name="author" content="Jake Yeung"> <meta name="description" content="Jake Yeung academic webpage. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://jakeyeung.github.io/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?0afe9f0ae161375728f7bcc5eb5b4ab4"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Jake</span> Yeung </h1> <p class="desc">Analytical Methods and Computational Biology</p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/prof_pic-480.webp 480w,/assets/img/prof_pic-800.webp 800w,/assets/img/prof_pic-1400.webp 1400w," sizes="(min-width: 800px) 231.0px, (min-width: 576px) 30vw, 95vw" type="image/webp"> <img src="/assets/img/prof_pic.jpg?75c81c059199c1ec4c8bc0b93d88096b" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> <div class="more-info"> <p>Principal Scientist.</p> <p>Genentech, Inc.</p> <p>1 DNA Way,</p> <p>South San Francisco, CA 94080, USA</p> </div> </div> <div class="clearfix"> <h1 id="overview">Overview</h1> <p>I am a computational biologist working at the interface of statistical/machine learning, functional genomics technologies, and genome biology. I focus on understanding how different cells turn/off different parts of our genome to give rise to different functions in our body, and how this control fails in disease.</p> <p>For my training, I was a Human Frontier Science Program (HFSP) fellow with Alexander van Oudenaarden at the Hubrecht Institute. Before that, I completed my PhD at the EPFL in computational systems biology advised by Felix Naef.</p> <h1 id="research-themes">Research themes</h1> <p>My projects span three approaches: computational methods, technology development, and biological function. They often pairs two of these approaches at a time (Fig. 1), bringing in wet and dry lab components.</p> <figure> <img src="/assets/img/triangle2.png" alt="Three_Approaches" width="300"> <figcaption><strong>Fig. 1: </strong>Blending the three approaches for diagnostics and therapeutics</figcaption> </figure> <h2 id="1-computationally-driven-experimental-design">(1) Computationally-driven experimental design</h2> <p>Computation ↔technology development.</p> <p>Inferring new information from single-cell experiments allow us to ask new questions about how cells function. Many advances have come from purely wet lab-driven approaches, but can hit diminishing returns. We have found developing new statistical methods alongside with new technologies can advance measurement limitations by inferring hidden biological parameters.</p> <p>Recently, we have recently developed a statistical unmixing method (scChIX-seq) that maps multiple histone modifications in single cells and learns the correlation structure between histone marks (<a href="https://www.nature.com/articles/s41587-022-01560-3" rel="external nofollow noopener" target="_blank">Yeung*, Florescu*, Zeller* et al 2023</a>). We are building on these multiplexing/unmixing methods to perform large-scale genetic and drug screening.</p> <figure> <img src="/assets/img/scchix_example.png" alt="scchix" width="1000"> <figcaption><strong>Fig. 2: </strong> Multiplexing and unmixing reveal relationships between chromatin states </figcaption> </figure> <h2 id="2-principles-of-data-analysis-for-new-functional-genomics-technologies">(2) Principles of data analysis for new functional genomics technologies</h2> <p>Tech dev ↔ biological function.</p> <p>New functional genomics technologies can now measure the regulation and output of every gene in the genome in virtually every cell in the body. However, the analysis side is challenging because it is unclear (a) what are the most exciting biological questions you can ask from different technolgoies given a fixed budget, (b) how to think about the design and analysis to separate biological signal from unwanted technical artifacts in sparse, noisy, and limited data regimes, and (c) how to integrate external information from other experiments done within an institute (with similar equipment, protocols, and staff) and around the world (where many conditions may differ) to improve statistical inference and prediction.</p> <p>Recently, we have shown that combining cell surface markers with histone modification mapping in single cells (sortChIC) can systematically compare the genome-wide dfferences between active (e.g. H3K4me1, H3K4me3) and repressed (e.g. H3K27me3, H3K9me3) chromatin states during blood maturation(<a href="https://www.nature.com/articles/s41588-022-01260-3" rel="external nofollow noopener" target="_blank">Zeller*, Yeung* et al 2022</a>). We are now developing new ways to infer time information and integrate that in the analysis.</p> <figure> <img src="/assets/img/sortchic_example.png" alt="sortChIC" width="1000"> <figcaption><strong>Fig. 3: </strong>Analysis methods that quantify global differences between cell types across chromatin states</figcaption> </figure> <h2 id="3-asking-how-cells-regulate-their-genomes">(3) Asking how cells regulate their genomes</h2> <p>Computation ↔ biological function.</p> <p>Although functional genomics technologies can now measure the expression and accessibility of thousands of genes across thousands of cells, discovering regulatory principles that explain the changes in gene expression over time and in different cell types is still challenging. Even when using established technologies, the analysis side remains the bottleneck because one must still refine what are the interesting questions to ask given a specific experiment, and craft the analysis to systematically tackle the question that will inspire the next incisive experiment. Possible principles are hypothesized through data exploration, then articulated in a statistical model, and tested against carefully crafted null hypotheses. At each step one must reason about the outputs in the context of how and why the experiment was performed.</p> <p>Recently, we have crafted model selection approaches to systematically categorize genes into a handful of groups that share similar regulation (<a href="https://genome.cshlp.org/content/28/2/182.full" rel="external nofollow noopener" target="_blank">Yeung*, Mermet* et al 2018</a>, <a href="https://www.pnas.org/doi/abs/10.1073/pnas.1910590116" rel="external nofollow noopener" target="_blank">Hor*, Yeung* et al 2019</a>). These methods allow the diverse gene expression patterns in a biological tissue to be explained in terms of a small number of biological regulatory principles, which leads to direct predictions that can be experimentally tested (<a href="https://genesdev.cshlp.org/content/32/5-6/347.full" rel="external nofollow noopener" target="_blank">Mermet*, Yeung* et al 2018</a>).</p> <figure> <img src="/assets/img/4cseq_dynamics.gif" alt="4Cseq" width="1000"> <figcaption><strong>Fig. 4: </strong>Changes in chromatin conformation at Cry1 over 24 hours</figcaption> </figure> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Jake Yeung. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. Last updated: February 15, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>