<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd"> <html><body> <h1 id="overview">Overview</h1> <p>I am a computational biologist working at the interface of statistical/machine learning, functional genomics technologies, and genome biology. I focus on understanding how different cells turn/off different parts of our genome to give rise to different functions in our body, and how this control fails in disease.</p> <p>For my training, I was a Human Frontier Science Program (HFSP) fellow with Alexander van Oudenaarden at the Hubrecht Institute. Before that, I completed my PhD at the EPFL in computational systems biology advised by Felix Naef.</p> <h1 id="research-themes">Research themes</h1> <p>My projects span three approaches (1) computational methods, (2) technology development, and (3) biological function. They often blend two of these approaches at a time, bringing in wet and dry lab components.</p> <p><img src="images/triangle2.png" alt="Three approaches to biological discovery"></p> <h2 id="computationally-driven-experimental-design">Computationally-driven experimental design</h2> <p>Computation ↔technology development.</p> <p>Being able to infer new information from single-cell experiments often allows us to ask new questions about how cells function. Purely wet lab-driven approaches to measuring new information can hit diminishing returns.</p> <p>Recently, we have recently developed a statistical unmixing method (scChIX-seq) that maps multiple histone modifications in single cells and learns the correlation structure between histone marks (<a href="https://www.nature.com/articles/s41587-022-01560-3" rel="external nofollow noopener" target="_blank">Yeung*, Florescu*, Zeller* et al 2023</a>). We are building on these multiplexing/unmixing methods to perform large-scale genetic and drug screening.</p> <p><img src="images/scchix_example.png" alt="Multiplexing and unmixing to uncover relationships between chromatin states"></p> <h1 id="many-advances-in-our-understanding-of-how-different-cells-in-our-body-can-turn-onoff-different-parts-of-the-genome-come-from">Many advances in our understanding of how different cells in our body can turn on/off different parts of the genome come from</h1> <h2 id="principles-of-data-analysis-for-new-functional-genomics-technologies">Principles of data analysis for new functional genomics technologies</h2> <p>Tech dev ↔ biological function.</p> <p>New functional genomics technologies can now measure the regulation and output of every gene in the genome in virtually every cell in the body. However, the analysis side is challenging because it is unclear what are the most exciting biological questions you can ask from each technology separately or from multiple technologies combined, and how to think about the design and analysis to separate biological signal from unwanted technical artifacts.</p> <p>Recently, we have shown that combining cell surface markers with histone modification mapping in single cells (sortChIC) can systematically compare the genome-wide dfferences between active (e.g. H3K4me1, H3K4me3) and repressed (e.g. H3K27me3, H3K9me3) chromatin states during blood maturation(<a href="https://www.nature.com/articles/s41588-022-01260-3" rel="external nofollow noopener" target="_blank">Zeller*, Yeung* et al 2022</a>). We are now developing new ways to infer time information and integrate that in the analysis.</p> <p><img src="images/sortchic_example.png" alt="Analyzing global differences across chromatin states"></p> <h2 id="asking-how-cells-regulate-their-genomes">Asking how cells regulate their genomes</h2> <p>Computation ↔ biological function.</p> <p>Although functional genomics technologies can now measure the expression and accessibility of thousands of genes across thousands of cells, discovering regulatory principles that explain the changes in gene expression over time and in different cell types is still challenging. Possible principles are hypothesized through data exploration, then articulated in a statistical model, and tested against carefully crafted null hypotheses.</p> <p>Recently, we have crafted model selection approaches to systematically categorize genes into a handful of groups that share similar regulation (<a href="https://genome.cshlp.org/content/28/2/182.full" rel="external nofollow noopener" target="_blank">Yeung*, Mermet* et al 2018</a>, <a href="https://www.pnas.org/doi/abs/10.1073/pnas.1910590116" rel="external nofollow noopener" target="_blank">Hor*, Yeung* et al 2019</a>). These methods allow the diverse gene expression patterns in a biological tissue to be explained in terms of a small number of biological regulatory principles, which leads to direct predictions that can be experimentally tested (<a href="https://genesdev.cshlp.org/content/32/5-6/347.full" rel="external nofollow noopener" target="_blank">Mermet*, Yeung* et al 2018</a>).</p> <p><img src="images/4cseq_dynamics.gif" alt="Changes in chromatin conformation at _Cry1_ over 24 hours"></p> </body></html>