<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Jake Yeung </title> <meta name="author" content="Jake Yeung"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://jakeyeung.github.io/publications/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?0afe9f0ae161375728f7bcc5eb5b4ab4"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Jake </span> Yeung </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="yeung2021deconvolving" class="col-sm-8"> <div class="title">scChIX-seq infers dynamic relationships between histone modifications in single cells</div> <div class="author"> Jake* Yeung ,  Maria Florescu* ,  Peter Zeller* , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Buys Anton Barbanson, Max D Wellenstein, Alexander Oudenaarden' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Nature Biotechnology</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="zeller2021hierarchical" class="col-sm-8"> <div class="title">Single-cell sortChIC identifies hierarchical chromatin dynamics during hematopoiesis</div> <div class="author"> Peter* Zeller ,  <em>Jake Yeung*</em> ,  Buys Anton Barbanson , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Helena Viñas Gaza, Maria Florescu, Alexander Oudenaarden' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Nature Genetics</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="yeung2022learning" class="col-sm-8"> <div class="title">Learning relationships between histone modifications in single cells</div> <div class="author"> <em>Jake Yeung</em> ,  Maria Florescu ,  Peter Zeller , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Buys Anton Barbanson, Max D Wellenstein, Alexander Oudenaarden' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>In NeurIPS 2022 Workshop on Learning Meaningful Representations of Life</em> , 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="phillips2021circadian" class="col-sm-8"> <div class="title">The circadian oscillator analysed at the single-transcript level</div> <div class="author"> Nicholas E Phillips ,  Alice Hugues ,  <em>Jake Yeung</em> , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Eric Durandau, Damien Nicolas, Felix Naef' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Molecular systems biology</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="mermet2021oscillating" class="col-sm-8"> <div class="title">Oscillating and stable genome topologies underlie hepatic physiological rhythms during the circadian cycle</div> <div class="author"> Jérôme* Mermet ,  <em>Jake Yeung*</em> ,  and  Felix Naef </div> <div class="periodical"> <em>PLoS Genetics</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Hor2019" class="col-sm-8"> <div class="title">Sleep–wake-driven and circadian contributions to daily rhythms in gene expression and chromatin accessibility in the murine cortex</div> <div class="author"> Charlotte N. Hor* ,  <em>Jake Yeung*</em> ,  Maxime Jan , and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Yann Emmenegger, Jeffrey Hubbard, Ioannis Xenarios, Felix Naef, Paul Franken' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences of the United States of America</em>, Dec 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The timing and duration of sleep results from the interaction between a homeostatic sleep–wake-driven process and a periodic circadian process, and involves changes in gene regulation and expression. Unraveling the contributions of both processes and their interaction to transcriptional and epigenomic regulatory dynamics requires sampling over time under conditions of unperturbed and perturbed sleep. We profiled mRNA expression and chromatin accessibility in the cerebral cortex of mice over a 3-d period, including a 6-h sleep deprivation (SD) on day 2. We used mathematical modeling to integrate time series of mRNA expression data with sleep–wake history, which established that a large proportion of rhythmic genes are governed by the homeostatic process with varying degrees of interaction with the circadian process, sometimes working in opposition. Remarkably, SD caused long-term effects on gene-expression dynamics, outlasting phenotypic recovery, most strikingly illustrated by a damped oscillation of most core clock genes, including Arntl/Bmal1, suggesting that enforced wakefulness directly impacts the molecular clock machinery. Chromatin accessibility proved highly plastic and dynamically affected by SD. Dynamics in distal regions, rather than promoters, correlated with mRNA expression, implying that changes in expression result from constitutively accessible promoters under the influence of enhancers or repressors. Serum response factor (SRF) was predicted as a transcriptional regulator driving immediate response, suggesting that SRF activity mirrors the build-up and release of sleep pressure. Our results demonstrate that a single, short SD has long-term aftereffects at the genomic regulatory level and highlights the importance of the sleep–wake distribution to diurnal rhythmicity and circadian processes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Weger2019" class="col-sm-8"> <div class="title">The Mouse Microbiome Is Required for Sex-Specific Diurnal Rhythms of Gene Expression and Metabolism</div> <div class="author"> Benjamin D. Weger ,  Cédric Gobet ,  <em>Jake Yeung</em> , and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Eva Martin, Sonia Jimenez, Bertrand Betrisey, Francis Foata, Bernard Berger, Aurélie Balvay, Anne Foussier, Aline Charpagne, Brigitte Boizet-Bonhoure, Chieh Jason Chou, Felix Naef, Frédéric Gachon' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Cell Metabolism</em>, Feb 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The circadian clock and associated feeding rhythms have a profound impact on metabolism and the gut microbiome. To what extent microbiota reciprocally affect daily rhythms of physiology in the host remains elusive. Here, we analyzed transcriptome and metabolome profiles of male and female germ-free mice. While mRNA expression of circadian clock genes revealed subtle changes in liver, intestine, and white adipose tissue, germ-free mice showed considerably altered expression of genes associated with rhythmic physiology. Strikingly, the absence of the microbiome attenuated liver sexual dimorphism and sex-specific rhythmicity. The resulting feminization of male and masculinization of female germ-free animals is likely caused by altered sexual development and growth hormone secretion, associated with differential activation of xenobiotic receptors. This defines a novel mechanism by which the microbiome regulates host metabolism.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Gachon2018" class="col-sm-8"> <div class="title">Cross-regulatory circuits linking inflammation, high-fat diet, and the circadian clock</div> <div class="author"> Frédéric Gachon ,  <em>Jake Yeung</em> ,  and  Felix Naef </div> <div class="periodical"> <em>Genes and Development</em>, Nov 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Mammalian physiology resonates with the daily changes in the external environment, allowing processes such as rest–activity cycles, metabolism, and body temperature to synchronize with daily changes in the surroundings. Studies have identified the molecular underpinnings of robust oscillations in gene expression occurring over the 24-h day, but how acute or chronic perturbations modulate gene expression rhythms, physiology, and behavior is still relatively unknown. In this issue of Genes &amp; Development, Hong and colleagues (pp. 1367–1379) studied how acute and chronic inflammation interacts with the circadian clock. They found that NF-\kappaB signaling can modify chromatin states and modulate expression of genes in the core clock network as well as circadian loco-motor behavior. Interestingly, a high-fat diet (HFD) fed to mice also triggers this inflammation pathway, suggesting that cross-regulatory circuits link inflammation, HFD, and the circadian clock.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Wang2018" class="col-sm-8"> <div class="title">Circadian clock-dependent and -independent posttranscriptional regulation underlies temporal mRNA accumulation in mouse liver.</div> <div class="author"> Jingkui Wang ,  Laura Symul ,  <em>Jake Yeung</em> , and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Cédric Gobet, Jonathan Sobel, Sarah Lück, Pål O Westermark, Nacho Molina, Felix Naef' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences of the United States of America</em>, Feb 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The mammalian circadian clock coordinates physiology with environmental cycles through the regulation of daily oscillations of gene expression. Thousands of transcripts exhibit rhythmic accumulations across mouse tissues, as determined by the balance of their synthesis and degradation. While diurnally rhythmic transcription regulation is well studied and often thought to be the main factor generating rhythmic mRNA accumulation, the extent of rhythmic posttranscriptional regulation is debated, and the kinetic parameters (e.g., half-lives), as well as the underlying regulators (e.g., mRNA-binding proteins) are relatively unexplored. Here, we developed a quantitative model for cyclic accumulations of pre-mRNA and mRNA from total RNA-seq data, and applied it to mouse liver. This allowed us to identify that about 20% of mRNA rhythms were driven by rhythmic mRNA degradation, and another 15% of mRNAs regulated by both rhythmic transcription and mRNA degradation. The method could also estimate mRNA half-lives and processing times in intact mouse liver. We then showed that, depending on mRNA half-life, rhythmic mRNA degradation can either amplify or tune phases of mRNA rhythms. By comparing mRNA rhythms in wild-type and Bmal1-/- animals, we found that the rhythmic degradation of many transcripts did not depend on a functional BMAL1. Interestingly clock-dependent and -independent degradation rhythms peaked at distinct times of day. We further predicted mRNA-binding proteins (mRBPs) that were implicated in the posttranscriptional regulation of mRNAs, either through stabilizing or destabilizing activities. Together, our results demonstrate how posttranscriptional regulation temporally shapes rhythmic mRNA accumulation in mouse liver.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Yeung2018Review" class="col-sm-8"> <div class="title">Rhythms of the Genome: Circadian Dynamics from Chromatin Topology, Tissue-Specific Gene Expression, to Behavior</div> <div class="author"> <em>Jake Yeung</em> ,  and  Felix Naef </div> <div class="periodical"> <em>Trends in Genetics</em>, Dec 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Circadian rhythms in physiology and behavior evolved to resonate with daily cycles in the external environment. In mammals, organs orchestrate temporal physiology over the 24-h day, which requires extensive gene expresion rhythms targeted to the right tissue. Although a core set of gene products oscillates across virtually all cell types, gene expression profiling across tissues over the 24-h day showed that rhythmic gene expression programs are tissue specific. We highlight recent progress in uncovering how the circadian clock interweaves with tissue-specific gene regulatory networks involving functions such as xenobiotic metabolism, glucose homeostasis, and sleep. This progress hinges on not only comprehensive experimental approaches but also computational methods for multivariate analysis of periodic functional genomics data. We emphasize dynamic chromatin interactions as a novel regulatory layer underlying circadian gene transcription, core clock functions, and ultimately behavior. Finally, we discuss perspectives on extending the knowledge of the circadian clock in animals to human chronobiology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Yeung2018" class="col-sm-8"> <div class="title">Transcription factor activity rhythms and tissue-specific chromatin interactions explain circadian gene expression across organs.</div> <div class="author"> Jake* Yeung ,  Jérôme Mermet* ,  Céline Jouffe , and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Julien Marquis, Aline Charpagne, Frédéric Gachon, Felix Naef' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Genome research</em>, Feb 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Temporal control of physiology requires the interplay between gene networks involved in daily timekeeping and tissue function across different organs. How the circadian clock interweaves with tissue-specific transcriptional programs is poorly understood. Here, we dissected temporal and tissue-specific regulation at multiple gene regulatory layers by examining mouse tissues with an intact or disrupted clock over time. Integrated analysis uncovered two distinct regulatory modes underlying tissue-specific rhythms: tissue-specific oscillations in transcription factor (TF) activity, which were linked to feeding-fasting cycles in liver and sodium homeostasis in kidney; and colocalized binding of clock and tissue-specific transcription factors at distal enhancers. Chromosome conformation capture (4C-seq) in liver and kidney identified liver-specific chromatin loops that recruited clock-bound enhancers to promoters to regulate liver-specific transcriptional rhythms. Furthermore, this looping was remarkably promoter-specific on the scale of less than 10 kilobases (kb). Enhancers can contact a rhythmic promoter while looping out nearby nonrhythmic alternative promoters, confining rhythmic enhancer activity to specific promoters. These findings suggest that chromatin folding enables the clock to regulate rhythmic transcription of specific promoters to output temporal transcriptional programs tailored to different tissues.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Mermet2018" class="col-sm-8"> <div class="title">Clock-dependent chromatin topology modulates circadian transcription and behavior.</div> <div class="author"> Jérôme* Mermet ,  <em>Jake Yeung*</em> ,  Clémence Hurni , and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Daniel Mauvoisin, Kyle Gustafson, Céline Jouffe, Damien Nicolas, Yann Emmenegger, Cédric Gobet, Paul Franken, Frédéric Gachon, Félix Naef' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Genes &amp; development</em>, Mar 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The circadian clock in animals orchestrates widespread oscillatory gene expression programs, which underlie 24-h rhythms in behavior and physiology. Several studies have shown the possible roles of transcription factors and chromatin marks in controlling cyclic gene expression. However, how daily active enhancers modulate rhythmic gene transcription in mammalian tissues is not known. Using circular chromosome conformation capture (4C) combined with sequencing (4C-seq), we discovered oscillatory promoter-enhancer interactions along the 24-h cycle in the mouse liver and kidney. Rhythms in chromatin interactions were abolished in arrhythmicBmal1knockout mice. Deleting a contacted intronic enhancer element in theCryptochrome 1(Cry1) gene was sufficient to compromise the rhythmic chromatin contacts in tissues. Moreover, the deletion reduced the daily dynamics ofCry1transcriptional burst frequency and, remarkably, shortened the circadian period of locomotor activity rhythms. Our results establish oscillating and clock-controlled promoter-enhancer looping as a regulatory layer underlying circadian transcription and behavior.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Mermet2016" class="col-sm-8"> <div class="title">Systems Chronobiology: Global Analysis of Gene Regulation in a 24-Hour Periodic World.</div> <div class="author"> Jérôme Mermet ,  <em>Jake Yeung</em> ,  and  Felix Naef </div> <div class="periodical"> <em>Cold Spring Harbor perspectives in biology</em>, Dec 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Mammals have evolved an internal timing system, the circadian clock, which synchronizes physiology and behavior to the daily light and dark cycles of the Earth. The master clock, located in the suprachiasmatic nucleus (SCN) of the brain, takes fluctuating light input from the retina and synchronizes other tissues to the same internal rhythm. The molecular clocks that drive these circadian rhythms are ticking in nearly all cells in the body. Efforts in systems chronobiology are now being directed at understanding, on a comprehensive scale, how the circadian clock controls different layers of gene regulation to provide robust timing cues at the cellular and tissue level. In this review, we introduce some basic concepts underlying periodicity of gene regulation, and then highlight recent genome-wide investigations on the propagation of rhythms across multiple regulatory layers in mammals, all the way from chromatin conformation to protein accumulation.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Lanen2015" class="col-sm-8"> <div class="title">Economic and environmental analysis of a green energy hub with energy storage under fixed and variable pricing structures</div> <div class="author"> Daniel Van Lanen ,  Jennifer Cocking ,  Sean B. Walker , and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Michael Fowler, Roydon Fraser, Steven B. Young, Leila Ahmadi, Alan Thai, Jake Yeung, Arthur Yip' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>International Journal of Process Systems Engineering</em>, Dec 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>With the increased use of intermittent renewable power generation, including wind and solar, the need for energy storage is increasing. Re-purposed hybrid electric vehicle lithium ion batteries have been shown to have energy storage potential at a reduced financial cost. In a two stage MatLAB simulation, a ‘Net Zero’ grid connected facility is created, and then the yearly electricity price of is calculated under various scenario using two different pricing structures, fixed Feed-in Tariff (FIT) and market. The facility under consideration is a commercial distribution center with refrigeration, onsite generation of hydrogen for fuel cell powered forklifts, solar and wind power generation, and re-purposed batteries for energy storage. The FIT mechanism is shown to be a deterrent to implementing energy storage onsite, as well as to increasing the use of locally generated power. The mechanism also negatively influences the indirect electricity emissions, which is not consistent with environmental objectives.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Wyatt2014" class="col-sm-8"> <div class="title">Heterogeneity in the inter-tumor transcriptome of high risk prostate cancer</div> <div class="author"> Alexander W Wyatt ,  Fan Mo ,  Kendric Wang , and <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Brian McConeghy, Sonal Brahmbhatt, Lina Jong, Devon M Mitchell, Rebecca L Johnston, Anne Haegert, Estelle Li, Janet Liew, Jake Yeung, Raunak Shrestha, Anna V Lapuk, Andrew McPherson, Robert Shukin, Robert H Bell, Shawn Anderson, Jennifer Bishop, Antonio Hurtado-Coll, Hong Xiao, Arul M Chinnaiyan, Rohit Mehra, Dong Lin, Yuzhuo Wang, Ladan Fazli, Martin E Gleave, Stanislav V Volik, Colin C Collins' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span> </div> <div class="periodical"> <em>Genome Biology</em>, Aug 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Genomic analyses of hundreds of prostate tumors have defined a diverse landscape of mutations and genome rearrangements, but the transcriptomic effect of this complexity is less well understood, particularly at the individual tumor level. We selected a cohort of 25 high-risk prostate tumors, representing the lethal phenotype, and applied deep RNA-sequencing and matched whole genome sequencing, followed by detailed molecular characterization. Ten tumors were exposed to neo-adjuvant hormone therapy and expressed marked evidence of therapy response in all except one extreme case, which demonstrated early resistance via apparent neuroendocrine transdifferentiation. We observe high inter-tumor heterogeneity, including unique sets of outlier transcripts in each tumor. Interestingly, outlier expression converged on druggable cellular pathways associated with cell cycle progression, translational control or immune regulation, suggesting distinct contemporary pathway affinity and a mechanism of tumor stratification. We characterize hundreds of novel fusion transcripts, including a high frequency of ETS fusions associated with complex genome rearrangements and the disruption of tumor suppressors. Remarkably, several tumors express unique but potentially-oncogenic non-ETS fusions, which may contribute to the phenotype of individual tumors, and have significance for disease progression. Finally, one ETS-negative tumor has a striking tandem duplication genotype which appears to be highly aggressive and present at low recurrence in ETS-negative prostate cancer, suggestive of a novel molecular subtype. The multitude of rare genomic and transcriptomic events detected in a high-risk tumor cohort offer novel opportunities for personalized oncology and their convergence on key pathways and functions has broad implications for precision medicine.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Jake Yeung. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. Last updated: February 13, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>